### PR TITLE
Added Kaley's profile picture and information for the lab website.

### PR DESCRIPTION
**Pull request by kaley-jenney**

### CODE REVIEW

File: docs/_data/team_members.yml [hunk 1]
```diff
@@ -135,3 +135,10 @@
   number_educ: 2
   education1: MSc. in Statistics, ETH Zurich
   education2: B.A. in Economics, University of St. Gallen
+  
+ - name: Kaley Jenney
+  photo: ../images/kaley.jpg
+  info: Technical Associate I
+  email: jenneyk@mit.edu
+  number_educ: 1
+  education1: B.S. in Neuroscience, Emmanuel College
```
Comment (satra) [inline docs/_data/team_members.yml:144]: you may want to update these lines by one more space.

Comment (kaley-jenney) [reply docs/_data/team_members.yml:144]: Can you confirm what you mean by this? Thank you!

Comment (kaley-jenney) [reply docs/_data/team_members.yml:144]: I just updated the lines by one more space.

File: docs/_data/team_members.yml [hunk 2]
```diff
@@ -135,3 +135,10 @@
   number_educ: 2
   education1: MSc. in Statistics, ETH Zurich
   education2: B.A. in Economics, University of St. Gallen
+  
+ - name: Kaley Jenney
```
Comment (satra) [inline docs/_data/team_members.yml:139]:
```suggestion
- name: Kaley Jenney
```

**Pull request merged**